tools/wpt/expectation.json

```diff
@@ -1014,7 +1014,6 @@
         "Response interface: operation json()",
         "Response interface: operation text()",
         "Response interface: calling redirect(USVString, optional unsigned short) on new Response() with too few arguments must throw TypeError",
-        "Window interface: calling fetch(RequestInfo, optional RequestInit) on window with too few arguments must throw TypeError",
```

Comment:
Just to confirm, this file is what is expected to fail? Therefore, by removing, it is expected to pass now, right?

Reply:
Yes. Looks good